src/adagrams.js

```diff
@@ -1,8 +1,113 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const pool = {
+      A: 9,
+      B: 2,
+      C: 2,
+      D: 4,
+      E: 12,
+      F: 2,
+      G: 3,
+      H: 2,
+      I: 9,
+      J: 1,
+      K: 1,
+      L: 4,
+      M: 2,
+      N: 6,
+      O: 8,
+      P: 2,
+      Q: 1,
+      R: 6,
+      S: 4,
+      T: 6,
+      U: 4,
+      V: 2,
+      W: 2,
+      X: 1,
+      Y: 2,
+      Z: 1
+    }
+
+    // don't alter orginal array by creating new array and pushing letters into it
```

Comment:
Your strategy here was to not alter the `pool` array within this method, so as to never alter it. And your strategy works! :+1: However, because of variable scope, you should feel free to alter it, because `const pool` will be re-created every time you call this method `drawLetters()`